pkg/generators/forwarding/fluentd/generators_test.go

```diff
@@ -0,0 +1,235 @@
+package fluentd
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+	"text/template"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/generators"
+)
+
+func TestConfigGenerator_generatePipelineToOutputLabels(t *testing.T) {
```

Comment:
Golang tests don't use snake case. Propose to rename to: `TestGeneratePipelineToOutputLabels`. The test file `generators_test.go` is a good enough identifier w/o the need to duplicate information like in the `TestConfigGenerator` prefix.

Reply:
https://golang.org/pkg/testing/
```
Multiple example functions for a package/type/function/method may be provided by appending a distinct suffix to the name. The suffix must start with a lower-case letter.

func Example_suffix() { ... }
func ExampleF_suffix() { ... }
func ExampleT_suffix() { ... }
func ExampleT_M_suffix() { ... }
```
So I don't think it breaks go test naming style. It is widely used, ex: https://github.com/hashicorp/consul/blob/master/api/agent_test.go#L170